docs-ref-conceptual/azure-cli-alias-extension.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Azure CLI 2.0 Alias Extension
+description: Using the Azure CLI Alias Extension
+keywords: Azure CLI, Extensions, Alias
+author: aaroney
+ms.author: aaroney
+manager: routlaw
+ms.date: 03/14/2018
+ms.topic: article
+ms.prod: azure
+ms.technology: azure
+ms.devlang: azurecli
+ms.service: multiple
+---
+
+# Azure CLI Alias Extension
+
+The extension allows users to define their own custom command alias in the Azure CLI based on existing commands. The motivation behind this is to introduce a fast, concise, and customizable way to interact with the Azure CLI, simplifying workflow, as well as boosting the productivity of our customers.
+
+> [!Note]
+> The Alias Extension is in public preview: the features and the configuration file format are subject to change.
+
+## Installing the Alias Extension
+**2.0.28** is the minimum version of the Azure CLI that is compatible with the alias extension. To check the version of your CLI, simply run `az --version`. A detailed guide to updating your Azure CLI can be found [here](./install-azure-cli.md).
+
+After updating your Azure CLI to the latest version, you can run the following command to install the extension:
+```
+az extension add --name alias
+```
+
+To verify your installation:
+```
+az extension list --output table
+```
+
+The command above will output a list of installed extensions in your CLI:
+```
+ExtensionType    Name                       Version
+---------------  -------------------------  ---------
+whl              alias                      0.2.0
+```
+
+## Working with the Alias Extension
+To start creating your custom aliases, you will have to first create a configuration file named `alias` (without any file extension) under the `.azure` folder. `.azure` is a hidden folder under your home directory that stores CLI-related configurations and settings. To create an empty alias configuration file, you can run the following commands depending on your operating system:
+
+- macOS / Ubuntu / Azure Cloud Shell / Bash on Windows
+```
+touch $HOME/.azure/alias
+```
+- Windows (Command Prompt)
+```
+type nul > %HOMEPATH%/.azure/alias
+```
+
+All of your alias definitions will be stored in the alias configuration file. The configuration file is based on the INI configuration format and has the following additional rules when defining aliases:
+- Each alias has a section named "command", which contains the Azure CLI commands to which the alias maps.
+- Each alias can contain multiple commands in the command section.
```

Comment:
@twitchax Please give me some more information about this. I don't see any examples listed which contain multiple commands.

Reply:
@sptramer alias `list-vm` refers to `vm list -g myResourceGroup`. There might be some confusion here because multiple commands in this context is referring to `vm` and `list`, not multiple az commands.